### PR TITLE
Add Edge Debugging blog post

### PR DESCRIPTION
**Pull request by auchenberg**
Pending review.  Do not merge

### CODE REVIEW

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+Order: 2
```
Comment (gregvanl) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: To be at the top, use the highest existing Order number + 1.

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+TOCTitle: Microsoft Edge Debugging for VS Code
```
Comment (gregvanl) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: This will be too long for the TOC.  How about just "Microsoft Edge Debugging" or "Edge Browser Debugging".
"Edge Debugging" would be good if you think enough readers will understand.

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+PageTitle: Microsoft Edge Debugging for VS Code
+MetaDescription: Microsoft Edge Debugging for Visual Studio Code
+Date: 2016-04-22
+ShortDescription: As a part of our continuous effort to simplify the daily workflow for developers, we are happy to enable web developers to debug their code running inside Microsoft Edge, directly from the editor by introducing Microsoft Edge Debugging for Visual Studio Code.
+
+Author: Kenneth Auchenberg
+---
+
+# Introducing Chrome Debugging for VS Code
```
Comment (gregvanl) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: Copy and paste error "Introducing Edge Debugging..."
I would also remove the "for VS Code" as it makes the h1 header too long.

**Comment by auchenberg** [comment]
Thanks @gregvanl. Hereby fixed.

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+April 22, 2016 by [Andy Sterland](https://twitter.com/AndySterland) and [Kenneth Auchenberg](https://twitter.com/auchenberg)
+
+As a part of our continuous effort to simplify the daily workflow for developers, we are happy to enable web developers to debug their code running inside Microsoft Edge, directly from the editor by introducing Microsoft Edge Debugging for Visual Studio Code.
```
Comment (gregvanl) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: Maybe reword like "As a part of our continued effort to simply the daily workflow for web developers, we are happy to announce Microsoft Edge Debugging for Visual Studio Code.  With this extension, you can now debug your source code running inside Microsoft Edge directly from the VS Code editor."

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+Our new Edge Debugger, works is quite similar to [our Chrome debugger](https://code.visualstudio.com/blogs/2016/02/23/introducing-chrome-debugger-for-vs-code) which we introduced back in February, and technically it’s the same debugger running inside VS Code, as we have embraced the [Chrome Debugger protocol](https://developer.chrome.com/devtools/docs/debugger-protocol) with our new Edge Diagnostics Adaptor, which we have written in detail about on the [Edge blog]().
```
Comment (gregvanl) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: Our new Edge Debugger works similar to our ... which we introduced back in February.  Technically, its the same debugger running inside VS Code as we used the ... with our new Edge Diagnostics Adapter.  You can learn the details in our [Edge blog].

File: blogs/2016/04/22/edge-debugging-for-vs-code.md (hunk 3 above)
Comment (auchenberg) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: You are commenting on a outdated commit, heads up :)

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+To get started, you simply open the Command Palette (`kb(workbench.action.showCommands)`) inside VS Code and type `ext install` to run the `Extensions: Install Extension` command.  When the extension list appears, type 'edge' to filter the list and install the `Debugger for Edge` extension.  You'll then create a launch-configuration file which we explain in detail in our README [right here](https://github.com/Microsoft/vscode-edge-debug).
+
+## Supported features
+
+In this release, we support the following features:
+
+- Setting breakpoints, including within source files when source maps are enabled
+- TypeScript, via source maps
+- Debug stepping
+- Locals scope variables via the Code Locals pane
```
Comment (gregvanl) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: Use "VS Code" not "Code"
Is there a Locals pane?  Isn't it VARIABLES?
For VS Code UI elements, we like to use bold and match the casing.  For example **WATCH** panel

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+- Debugging eval scripts, script tags, and scripts that are added dynamically
+- Watches via the VS Code Watch panel.
+- The debug console
+- Most console APIs
+
+## Going forward
+
+If you have any issues or ideas for improvements, feel free to reach out to us on [Twitter](https://twitter.com/code) or [GitHub](https://github.com/Microsoft/vscode-edge-debug/).
```
Comment (gregvanl) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: Did you mean to use the VS Code twitter account?

Comment (auchenberg) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: Yup, it's the balance of choosing a "shipping identity". This time it's "VSCode" that's the sender.

File: blogs/2016/04/22/edge-debugging-for-vs-code.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+# Introducing Microsoft Edge Debugging for VS Code
+
+April 22, 2016 by [Andy Sterland](https://twitter.com/AndySterland) and [Kenneth Auchenberg](https://twitter.com/auchenberg)
+
+As a part of our continuous effort to simplify the daily workflow for developers, we are happy to enable web developers to debug their code running inside Microsoft Edge, directly from the editor by introducing Microsoft Edge Debugging for Visual Studio Code.
+
+Our new Edge Debugger works similar to [our Chrome debugger](https://code.visualstudio.com/blogs/2016/02/23/introducing-chrome-debugger-for-vs-code) which we introduced back in February. Technically it’s the same debugger running inside VS Code, as we are using the [Chrome Debugger protocol](https://developer.chrome.com/devtools/docs/debugger-protocol) with our new Edge Diagnostics Adaptor. 
+
+You can learn all the details about the Edge Diagnostics Adaptor on the [Edge blog](http://blogs.windows.com/msedgedev/2016/04/22/introducing-edge-diagnostics-adapter/).
```
Comment (bootstraponline) [inline blogs/2016/04/22/edge-debugging-for-vs-code.md]: > http://blogs.windows.com/msedgedev/2016/04/22/introducing-edge-diagnostics-adapter/

^ that link is broken... should be:

> https://blogs.windows.com/msedgedev/2016/04/27/introducing-edge-diagnostics-adapter/

**Comment by gregvanl** [comment]
Closing the pull request as this blog post is hosted at https://blogs.windows.com/msedgedev/2016/04/27/introducing-edge-diagnostics-adapter/
